### PR TITLE
docs: fix various typos

### PR DESCRIPTION
**Pull request by luzpaz**
## Description

Various user-facing and non-user-facing typo fixes.

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [X] (**possibly?**) Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] (**possibly?**) This change requires a documentation update

## How Has This Been Tested?

n/a

## Checklist:

- [ ] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

<a href="https://gitpod.io/#https://github.com/fonoster/routr/pull/205"></a>

### CODE REVIEW

File: mods/dispatcher/src/types.ts [hunk 1]
```diff
@@ -42,7 +42,7 @@ export type CallbackErrors =
 
 export type ProcessorCallback = (
   err: CallbackErrors,
-  reponse?: MessageRequest
+  response?: MessageRequest
```
Comment (luzpaz) [inline mods/dispatcher/src/types.ts:45]: Please review (may be breaking change)

Comment (psanders) [reply mods/dispatcher/src/types.ts:45]: Hey @luzpaz 

Thanks for the note. 

I believe it will be ok since it's a positional parameter.

**Review by psanders — APPROVED** [review]
@luzpaz,

I greatly appreciate this pull request and will likely merge it next week 😊

**Comment by psanders** [comment]
@luzpaz  a huge thanks for submitting this PR. It means a lot!

(Q. Are you in the Discord community? I want to give you a shout out.)

**Comment by luzpaz** [comment]
I'm not. but thanks :pray:

**Comment by psanders** [comment]
@luzpaz Here is the link if you decide to join => https://discord.com/invite/4QWgSz4hTC

Thanks again.

**Pull request merged**